### PR TITLE
chore: Adding csi-proxy-v0.2.2 to Windows VHD

### LINKED ISSUE #1968: Add Windows CSI proxy to Windows-AKS VHDs
**Describe the request**

Windows support for CSI will require adding a new service. This is described in https://github.com/kubernetes/enhancements/blob/master/keps/sig-windows/20190714-windows-csi-support.md

**Explain why AKS Engine needs it**

It needs to be there for the out-of-tree Azure storage plugins to work

https://github.com/Azure/container-compute-upstream/issues/6

**Describe the solution you'd like**

It should be in the Windows AKS VHD (#1084) so it's preinstalled

**Describe alternatives you've considered**

Install CSI on host instead of using daemon sets. This means nodes need to be reprovisioned to get an updated CSI implementation.

**Additional context**

### PR DESCRIPTION
**Pull request by marosset**
**Reason for Change**:

**Issue Fixed**:

Fixes #1968 

**Credit Where Due**:

Does this change contain code from or inspired by another project?

- [x] No
- [ ] Yes

If "Yes," did you notify that project's maintainers and provide attribution?

- [ ] No
- [ ] Yes

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Review by mboersma — APPROVED** [review]
/lgtm

File: vhd/packer/configure-windows-vhd.ps1 [hunk 1]
```diff
@@ -116,7 +118,7 @@ function Get-FilesToCacheOnVHD {
             "https://kubernetesartifacts.azureedge.net/kubernetes/v1.18.10/windowszip/v1.18.10-1int.zip",
             "https://kubernetesartifacts.azureedge.net/kubernetes/v1.19.1/windowszip/v1.19.1-1int.zip",
             "https://kubernetesartifacts.azureedge.net/kubernetes/v1.19.2/windowszip/v1.19.2-1int.zip",
-            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.19.3/windowszip/v1.19.3-1int.zip"
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.19.3/windowszip/v1.19.3-1int.zip",
```
Comment (jackfrancis) [inline vhd/packer/configure-windows-vhd.ps1:121]: how was this not broken before?

Comment (marosset) [reply vhd/packer/configure-windows-vhd.ps1:121]: Good question

**Pull request merged**